docs/README.md

```diff
@@ -67,17 +65,16 @@ own.
 	The `docker/docker` repository contains a `Dockerfile` and a `Makefile`.
 	Together, these create a development environment in which you can build and
 	run a container running the Docker documentation website. To build the
-	documentation site, enter `make docs` at the root of your `docker/docker`
-	fork:
-	
+	documentation site, enter `make docs` in the `docs` directory of your `docker/docker` fork:
```

Comment:
Ah, thanks, yup that makes sense then (wasn't aware of the other repos being different there)